app/grandchallenge/reader_studies/models.py

```diff
@@ -1024,6 +1024,7 @@ def get_widget_required_types():
 class QuestionWidgetKindChoices(models.TextChoices):
     ACCEPT_REJECT = "ACCEPT_REJECT", "Accept/Reject Findings"
     NUMBER_INPUT = "NUMBER_INPUT", "Number input"
+    SLIDER = "Slider", "Slider"
```

Comment:
Also needs changing in the migration. Shouldn't we call it `Range` to match the html specs? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range

```suggestion
    SLIDER = "SLIDER", "Slider"
```

Reply:
I think NUMBER_RANGE is the best fit with the others. I'll update things.